include/cute/atom/copy_traits_xe.hpp

```diff
@@ -246,7 +246,7 @@ CUTE_HOST_DEVICE auto prefetch_selector(Tensor const& tensor) {
   using PrefetchOp = typename choose_prefetch_for_type<dtype_size_bits, block_non_contig_size>::Prefetch;
   using PrefetchTraits = Copy_Traits<PrefetchOp, decltype(tensor.stride())>;
   using PrefetchAtom = Copy_Atom<PrefetchTraits, dtype>;
-  using Scalar = Int<cute::max(1, 8 / dtype_size_bits)>;
+  using Scalar = Int<cute::max(1, detail::size_of_inst_bits<typename PrefetchTraits::CopyOp, int8_t> / dtype_size_bits)>;
```

Comment:
```suggestion
  using Scalar = Int<cute::max(1, detail::size_of_inst_bits<PrefetchOp, int8_t> / dtype_size_bits)>;
```
I think this is equivalent, right?